oberon-chat/chat-server
# Add backend support for starring messages

### Why: 
We would like chat users to be able to star messages and refer back to a history of starred messages.

### What: 
This PR lays the backend groundwork for associating a user with starred messages. 

It creates a join table between Users and Messages called SharedMessages, and adds a schema for SharedMessages with `create` and `delete functions`.

### Verification Steps:
See https://github.com/oberon-chat/chat-client/pull/4 for combined verification steps.
